src/calculator.ts

```diff
@@ -1,54 +1,86 @@
 import fs from "fs";
 import { linearQF, Contribution, Calculation } from "pluralistic";
+import {AugmentedResult, CalculatorOptions, RawContribution} from "./types.js";
```

Comment:
I think we should keep these types inside this file for now. If the responsibility of this file ever becomes unmanageable we can think about how to split it up in features, but not a generic `types.ts` file.

Reply:
Either way we need to start thinking on how to organize this codebase and split up the different modules in directories a little bit, for example of the top of my head: CLI, prices, blockchain indexer, calculator, http server.

Reply:
Agreed - this was my way of saying we should be thinking about this.  We can spike it down the line.